src/floating_label.cpp

```diff
@@ -180,7 +185,7 @@ void floating_label::update(int time)
 	// Invalidate new draw loc in preparation
 	draw_manager::invalidate_region(get_bg_rect(draw_loc));
 
-	DBG_FT << "updating floating label from " << screen_loc_ << " to " << draw_loc;
+//	DBG_FT << "updating floating label from " << screen_loc_ << " to " << draw_loc;
```

Comment:
Why did you comment this out? If you don't need it, just remove it.

Reply:
Oh.  I commented those out because they were spewing 60 messages each every second to the log I was trying to watch, and I just forgot to put them back.  Actually, I suspect someone else just forgot to remove them when they were no longer needed so they should probably be removed, but I'm hesitant to do since I'm not sure.  I can uncomment them or remove them. ?

Reply:
Uncomment them, since they are also uncommented in master.